Gemfile

```diff
@@ -3,28 +3,28 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '>= 2.6.5'
+ruby '>= 3.1.0'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 6.0.6'
+gem 'rails', '~> 6.1', '>= 6.1.7.3'
 # Use Puma as the app server
-gem 'puma', '~> 5.6'
+gem 'puma', '~> 6.2', '>= 6.2.1'
 
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 # gem 'jbuilder', '~> 2.7'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
 
 # Redis connection setup for live session (server and meeting) tracking
-gem 'connection_pool', '~> 2.3.0'
+gem 'connection_pool', '~> 2.4.0'
 gem 'redis', '~> 4.8.0'
 gem 'redis-namespace', '~> 1.9.0'
```

Comment:
Can this be updated to v1.10 to remove that annoying error?

Reply:
I thought I already did :facepalm: . I'll update it now.